pkgs/development/tools/turso-cli/default.nix

```diff
@@ -1,27 +1,35 @@
 {
   lib,
-  buildGoModule,
+  buildGo121Module,
   fetchFromGitHub,
 }:
-buildGoModule rec {
+buildGo121Module rec {
```

Comment:
After https://github.com/NixOS/nixpkgs/pull/254878 we should be able to get away with using `buildGoModule`. Are you able to double check the requirement to explicitly call `buildGo121Module`?

Reply:
When I started the branch `buildGoModule` was using 1.20

If that changed to 1.21 in the meantime than I can switch.

Reply:
Actually, I just realised that PR only went into staging so `buildGoModule` would still be using 1.20 afaik